src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -128481,6 +128447,42 @@ public new void M1<T>(System.Nullable<T?> x)
                 );
         }
 
+        [Fact, WorkItem(43071, "https://github.com/dotnet/roslyn/issues/43071")]
```

Comment:
Does the implementation change affect behavior of this test?

Reply:
No behavior change to this test, I'm just adding a test to close a different issue.